jmix-flowui/flowui/src/main/java/io/jmix/flowui/xml/layout/support/ComponentLoaderSupport.java

```diff
@@ -443,6 +421,44 @@ public void loadMetaClass(SupportsMetaClass component, Element element) {
                         component.setMetaClass(applicationContext.getBean(Metadata.class).getClass(metaClass)));
     }
 
+    public void loadDatePickerI18n(Element element, Consumer<DatePicker.DatePickerI18n> setter) {
+        DatePicker.DatePickerI18n datePickerI18n = new DatePicker.DatePickerI18n();
+
+        loadFirstDayOfWeek(datePickerI18n, element);
+        loadDateFormat(datePickerI18n, element);
+
+        setter.accept(datePickerI18n);
+    }
+
+    protected void loadDateFormat(DatePicker.DatePickerI18n datePickerI18n, Element element) {
```

Comment:
This method has become `protected`, do we accept such API changes?
It's kind of like more for internal use?

Reply:
So... will be marked as deprecated until the next minor release.